Web/stockdogweb/src/components/News/News.js

```diff
@@ -8,6 +8,11 @@ class News extends Component {
       const headlineObjs = this.props.headlines;
       var headlines = [];
 
+      // Don't do anything if there are no headlines
+      if (!headlineObjs) {
```

Comment:
Same comment as the listings above about whether or not it's null or just an empty array

Reply:
Same comment as above about PropTypes